spirv_common.hpp

```diff
@@ -107,7 +107,11 @@ inline std::string convert_to_string(float t)
 	// std::to_string for floating point values is broken.
 	// Fallback to something more sane.
 	char buf[64];
+
+	auto old = std::locale::global(std::locale::classic());
```

Comment:
According to [the docs](http://en.cppreference.com/w/cpp/locale/locale/global), std::local::global will set the C locale as well if the supplied locale has a name.